eslint.config.js

```diff
@@ -2,23 +2,18 @@ import babelParser from '@babel/eslint-parser';
 import js from '@eslint/js';
 import globals from 'globals';
 import noOnlyTests from 'eslint-plugin-no-only-tests';
+import eslintConfigPrettier from 'eslint-config-prettier/flat';
 import tseslint from 'typescript-eslint';
 
 export default [
   {
     // https://github.com/eslint/eslint/discussions/18304#discussioncomment-9069706
-    ignores: [
-      'node_modules/*',
-      'dist/*',
-      'coverage/*',
-    ],
+    ignores: ['node_modules/*', 'dist/*', 'coverage/*'],
   },
   {
     languageOptions: {
       parser: babelParser,
       parserOptions: {
-        ecmaVersion: 2022,
-        sourceType: 'module',
```

Comment:
Removed these lines because we want ecmaVersion to `latest`, and sourceType to `module` both of which is the default